src/custom-element.js

```diff
@@ -783,6 +782,10 @@ function createBaseAmpElementProto(win) {
    * @final @this {!Element}
    */
   ElementProto.attachedCallback = function() {
+    this.classList.add('-amp-element');
```

Comment:
Element can be attached any number of times. So, we should do this either:
 1. When the element attached the first time (see `this.everAttached`)
 2. When it's not built yet (see `this.built_`)
 3. Or both :)